administration/src/bp-modules/user-settings/ApiTokenSettings.tsx

```diff
@@ -115,6 +116,7 @@ const ApiTokenSetting = () => {
       </Alert>
       <SettingsCard>
         <H2>Api Token</H2>
+        <PepperSettings />
```

Comment:
I disagree. This pepper is directly linked to the api token. Using one without the other makes no sense, so they should be close in the UI. I changed the description from 
`Um den Endpunkt zu nutzen, müssen die Nutzerdaten mit folgenden Pepper gehashet werden:` to `Die Nutzerdaten müssen mit folgenden Pepper gehashed werden:` If you have a better suggestion or a strong wish for a wording here, let me know.

Reply:
For me this hashing information is still out of context for the user.

Well i mean the question is if you want a general api token area (if its used for other issues later) or use this since we don't have other use cases for a particular case like the user import.

If we focus on the user import my suggestions would be.

Use a NavBar item "User Import"
- show a short description of the purpose of the user import
- show the endpoint url
- show the hashing pepper with short explanation (as you provided in last comment)
- show API token part or display sth like "For authentification you need an api token -> get it here and add a link to user settings

i think we are currently mixing general things of the user import (not user related) and api-token related things that are used related. Maybe UI/UX can also asked for support

Reply:
api-tokens are not user related, they are user import releated. All project admins of koblenz can see all api-tokens of koblenz.

Reply:
ok, makes sense to move it to project settings